docs/source/channels.rst

```diff
@@ -0,0 +1,99 @@
+Channels: Futures Shared Between Clients
+===================================
```

Comment:
This bar should match the number of characters in the title

Rather than "Futures Shared" how about "Share Futures" to keep the voice active.